esp/Joint/src/as5600.h

```diff
@@ -0,0 +1,20 @@
+#ifndef AS5600_H
```

Comment:
is this your file or some official library? If your, can you please add extra comment what "AS5600" is? I guess some hall sensor(?) and the class provides info about conversion to degrees? I would maybe mention it with angle() function too as other may expect radians ;). Thanks